c7n/tags.py

```diff
@@ -72,11 +72,19 @@ def universal_augment(self, resources):
         return resources
 
     # For global resources, tags don't populate in the get_resources call
-    # unless the call is being made to us-east-1
-    region = getattr(self.resource_type, 'global_resource', None) and 'us-east-1' or self.region
-
+    # unless the call is being made to us-east-1. For govcloud this is us-gov-west-1.
+    partition = utils.get_partition(self.region)
+    r = ""
+    if partition == "aws-us-gov":
```

Comment:
this logic feels like it would be better in a utility function in utils.py, like `get_global_region(partition)` it would also help on the lack of test coverage